features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/RumMonitor.kt

```diff
@@ -283,6 +283,13 @@ interface RumMonitor {
         value: Any
     )
 
+    /**
+     * Adds result of evaluating a set of feature flag to the view.
+     * Feature flag evaluations are local to the active view and are cleared when the view is stopped.
+     * @param featureFlags the map of feature flags
+     */
+    fun addFeatureFlagBatchEvaluation(featureFlags: Map<String, Any>)
```

Comment:
maybe we should deprecate `addFeatureFlagEvaluation` method in favour of this one? And call this one also `addFeatureFlagEvaluation`?

Reply:
Both make sense IMO, they strike a balance between convenience (single flag) and performance (bulked flags). However, we could enhance the API comments in both to mention the performance benefit in batching FF evaluations.

My suggestion would be tho to name it `addFeatureFlagEvaluations` (plural). Keep the old one singular.